src/ExecBlock/ExecBlock.cpp

```diff
@@ -139,7 +140,7 @@ void ExecBlock::show() const {
         );
 
         llvm::raw_string_ostream disassOs(disass);
-        assembly.printDisasm(inst, disassOs);
+        assembly.printDisasm(inst, reinterpret_cast<uint64_t>(codeBlock.base()) + i, disassOs);
```

Comment:
Better to use ``uintptr_t`` instead of ``uint64_t``

Reply:
I prefer to keep a uint64_t to match the LLVM API (https://github.com/llvm/llvm-project/blob/llvmorg-10.0.1/llvm/include/llvm/MC/MCDisassembler/MCDisassembler.h#L77)